File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-jamba-development/iOS_Raleys/Raley's/Raley's/Classes/Models/Requests/EcartPreOrderResponse.m
//
//  EcartPreOrderResponse.m
//  Raley's
//
//  Created by Bill Lewis on 3/18/14.
//  Copyright (c) 2014 Raleys. All rights reserved.
//

#import "EcartPreOrderResponse.h"

@implementation EcartPreOrderResponse

@synthesize accountId;
@synthesize listId;
@synthesize sePoints;
@synthesize productPrice;
@synthesize salesTax;
@synthesize crv;
@synthesize fees;
@synthesize totalPrice;
@synthesize appointmentList;

- (NSString *)getComponentTypeForArray:(NSString *)propertyName
{
    if([propertyName isEqualToString:@"appointmentList"])
        return @"EcartAppointment";
    else
        return nil;
}

@end

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-jamba-development/Android_Raleys/Raleys/src/com/raleys/app/android/models/EcartPreOrderResponse.java
package com.raleys.app.android.models;

import java.util.ArrayList;

public class EcartPreOrderResponse
{
	public String accountId;
	public String listId;
	public int    sePoints;
	public float  productPrice;
	public float  salesTax;
	public float  crv;
	public float  fees;
	public float  totalPrice;
	public ArrayList<EcartAppointment> appointmentList;
}
